notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>With all the helpers function I find the overall architecture hard to follows. Is there a way to make all the code in this lab shorter and the structure more apparent? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Helper functions removed! Thanks